src/components/typography/typography.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+
+const H1 = styled.h1`
+  font-size: 32px;
+  line-height: 110%;
+
+  ${up('md')} {
+    font-size: 72px;
+  }
+`;
+
+const H2 = styled.h1`
+  font-size: 48px;
+  line-height: 110%;
+
+  ${up('md')} {
+    font-size: 72px;
+  }
+`;
+
+const Body = styled.div``;
+
+type Variant = 'h1' | 'h2';
+
+interface TypographyProps extends HTMLAttributes<{}> {
+  variant: Variant;
+}
+
+export const Typography: React.FC<TypographyProps> = (props) => {
```

Comment:
I don't think such a "Factory" makes sense. You could just import the according component instead.
What's the purpose of this?

Reply:
I took the typography component from material-ui as role model, but at this development stage it's probably simpler to make separate components each. Done.